pkg/proc/bininfo.go

```diff
@@ -503,6 +501,12 @@ func (image *Image) registerRuntimeTypeToDIE(entry *dwarf.Entry, ardr *reader.Re
 // the relocation offset) for all other images.
 // The first image added must be the executable file.
 func (bi *BinaryInfo) AddImage(path string, addr uint64) error {
+	if bi.lastModified.IsZero() {
```

Comment:
This is altering the semantics of bi.lastModified. It used to be the last modified time of the executable or zero if the executable can not be stat'ed. Now it is the last modified time of the first image we can stat. The old semantics is better.

Reply:
Ack, you're right. I'll try and find a better way.